ONSdigital/takeon-ui
# Feature/spp 830/display adjusted response in historic data

# Description
- Changed text for the tabs 
- Added historical data (adjusted tab
Story link: https://collaborate2.ons.gov.uk/jira/secure/RapidBoard.jspa?rapidView=1583&projectKey=SPP&view=detail&selectedIssue=SPP-830

# How to test?
- Ensure that the tabs can be seen on the view, edit and override screens
- Ensure the spelling is correct for the tabs

Env 830 has been used for development

# Links
Business Layer PR: https://github.com/ONSdigital/takeon-business-layer/pull/95
